site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -527,57 +552,55 @@ If successful, you should see a response that looks something like this:
     "preferredLanguages": [
       "en"
     ],
-    "verified": true,
+    "verified": True,
     "tenantId": "31626131-3938-6231-3634-663636656635",
-    "username": "terry@example.com",
-    "mobilePhone": "4245551212",
-    "passwordChangeRequired": false,
-    "fullName": "Terry Example",
+    "username": "jerryexample",
+    "passwordChangeRequired": False,
+    "fullName": "Jerry Example",
     "insertInstant": 1569515252886,
     "lastName": "Example",
-    "twoFactorEnabled": false,
-    "email": "terry@example.com"
+    "email": "jerry@example.com"
   }
 }
 ```
 
-Navigate to **Users** in the FusionAuth UI. You should now see a new user created and registered to an application!
-
-{% include _image.liquid src="/assets/img/blogs/manage-user-screen.png" alt="Manage User Screen" class="img-fluid full mb-4" figure=false %}
+Navigate to "**Users**" in the FusionAuth UI. You should now see a new user created and registered to an application!
 
+{% include _image.liquid src="/assets/img/blogs/implementing-fusionauth-python/add-user-api.png" alt="Manage User Screen" class="img-fluid full
+mb-4" figure=false %}
 
 ## Troubleshooting Common Errors
+
 What if your app(s) didn't quite work? Here's a list of common error codes that your app or server might return:
 
 ### Code 401
 
 **Description:** "_You did not supply a valid Authorization header. The header was omitted or your API key was not valid. The response will be empty. See [Authentication](/docs/v1/tech/apis/authentication)._"
 
-**What this usually means:** Your API key is likely invalid. Did you remember to replace the code example API key with the one from your own FusionAuth server? Did you copy your Application ID instead of your API key? Open up a web browser and go to **Settings > API Keys**, check your API key, and try again.
+**What this usually means:** Your API key is likely invalid. Did you remember to replace the code example API key with the one from your own FusionAuth server? Did you copy your Application Id instead of your API key? Open up a web browser and go to "**Settings**" and then "API Keys" to check your API key, and try again.
```

Comment:
```suggestion
**What this usually means:** Your API key is likely invalid. Did you remember to replace the code example API key with the one from your own FusionAuth server? Did you copy your Application Id instead of your API key? Open up a web browser and go to "Settings" and then "API Keys" to check your API key, and try again.
```

Reply:
missed one! thx